docs/diagrams/alias/AliasExecutionActivityDiagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+!include ../style.puml
+
+start
+:User enters a command;
+repeat :Parse command string;
+if () then ([begins with existing command word])
+:Parse as existing command;
+if () then ([all parameters are valid)
+:Return command;
```

Comment:
There may be a missing closing bracket here:

`if () then ([all parameters are valid)`

Reply:
Good catch!